File: apps/institution-api/Models/InstitutionLocationPhone.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace ProviderApi.Models
{
    public partial class InstitutionLocationPhone
    {
        public int InstitutionLocationPhoneId { get; set; }
        public int InstitutionLocationId { get; set; }
        public Guid PhoneGuid { get; set; }
        public string PhoneUsage { get; set; }
        public string PhoneType { get; set; }
        public int? ListSequence { get; set; }
        public string CountryCallingCode { get; set; }
        public string PhoneNumber { get; set; }
        public string Extension { get; set; }
        public DateTime? PhoneBusinessStartDate { get; set; }
        public DateTime? PhoneBusinessEndDate { get; set; }

        public virtual InstitutionLocation InstitutionLocation { get; set; }
    }
}

File: apps/planmp/src/libs/common/models/institution-location-phone.ts


import { InstitutionLocation } from './institution-location';

export class InstitutionLocationPhone {
    public institutionLocationPhoneId: number = 0;
    public institutionLocationId: number = 0;
    public phoneGuid: string = '';
    public phoneUsage: string = '';
    public phoneType: string = '';
    public listSequence: number;
    public countryCallingCode: string = '';
    public phoneNumber: string = '';
    public extension: string = '';
    public phoneBusinessStartDate: Date;
    public phoneBusinessEndDate: Date;
    public institutionLocation: InstitutionLocation = new InstitutionLocation();

    constructor(init? : Partial<InstitutionLocationPhone>) {
        
        Object.assign(this, init);
    }
}

